src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -940,6 +941,25 @@ public Map<Digest, Set<String>> getBlobDigestsWorkers(Iterable<Digest> blobDiges
     return state.casWorkerMap.getMap(client, blobDigests);
   }
 
+  @Override
+  public Map<String, Integer> updateCasReadCount(
+      Stream<Map.Entry<Digest, Integer>> casReadCountStream) throws IOException {
+    return client.call(
+        jedis ->
+            state.casReadCount.incrementMembersScore(
+                jedis,
+                casReadCountStream.map(
+                    entry ->
+                        new AbstractMap.SimpleEntry<>(
+                            entry.getKey().getHash(), entry.getValue()))));
```

Comment:
There are some Maps transformations that would be suitable in guava for this. The Map is looking like a better candidate for interface parameter typing as well.

Reply:
Changed it to traditional forEach. I wasn't able to find any method to transform keys in guava something like `Maps.transformKeys`.